v4/weaviate/data/objectsGetter.go

```diff
@@ -113,6 +135,13 @@ func (getter *ObjectsGetter) buildPathParams() string {
 		}
 	}
 
+	if getter.withConsistencyLevel {
+		pathParams = append(pathParams, fmt.Sprintf("consistency_level=%v", getter.consistencyLevel))
+	}
+	if getter.withNodeName {
+		pathParams = append(pathParams, fmt.Sprintf("node_name=%v", getter.nodeName))
+	}
```

Comment:
Is there any validation that only one of both is added?

Reply:
Yes, the server validates this